common/src/main/java/feast/common/models/Store.java

```diff
@@ -17,11 +17,35 @@
 package feast.common.models;
 
 import feast.proto.core.StoreProto.Store.Subscription;
+import java.util.Arrays;
 import java.util.List;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 public class Store {
 
+  /**
+   * Accepts a comma-delimited Subscriptions that is string-formatted and converts it to a list of
+   * Subscription class objects.
+   *
+   * @param subscriptions String formatted Subscriptions, comma delimited.
+   * @param exclude flag to determine if subscriptions with exclusion flag should be returned
+   * @return List of Subscription class objects
+   */
+  public static List<Subscription> parseSubscriptionFrom(String subscriptions, boolean exclude) {
```

Comment:
How about `parseSubscriptionFromStr(String subscriptions)` and `parseSubscriptionFromStrWithoutExclusions(String subscriptions)`